src/Tribe/Widget/Widget_Abstract.php

```diff
@@ -280,6 +350,13 @@ public function filter_argument( $argument, $index, $default = null  ) {
 	 * {@inheritDoc}
 	 */
 	public function get_default_arguments() {
+
+		// Setup admin fields.
+		$this->default_arguments['admin_fields'] = $this->get_admin_fields();
```

Comment:
I would move this to the `__construct` method; having a getter method be also the one that initializes property is confusing. Properties should be initialized in the constructor.
If that cannot be done, then this method should NOT modify the `default_arguments` property and should just read it, augment it in a variable and send that variable to the `filter_default_arguments` method.